mm2src/mm2_main/src/lp_ordermatch/my_orders_storage.rs

```diff
@@ -706,6 +706,8 @@ mod tests {
 
     wasm_bindgen_test_configure!(run_in_browser);
 
+    const LEGACY_SWAP_V: u32 = 1;
```

Comment:
`legacy_swap_version` can replace this need

Reply:
Done https://github.com/KomodoPlatform/komodo-defi-framework/commit/d4c5a16d6c3bdb70346e8111abf1b29ac0488c40